Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -408,6 +408,39 @@ application pods are now redirected to the ``node-local-dns`` pod.
 In the absence of a node-local DNS cache, DNS queries from application pods
 will get directed to cluster DNS pods backed by the ``kube-dns`` service.
 
+* Troubleshooting
```

Comment:
```suggestion
Troubleshooting
~~~~~~~~~~~~~~~
```
I'd expect the troubleshooting section to be a nested section, not just a bullet point. But given the deployment instructions are also bullet points, I won't block on it.

Reply:
Tried this, the troubleshooting section sort of merges in the block of text above. 

>But given the deployment instructions are also bullet points, I won't block on it.

Right! It's aligned to the other bullet points.